api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/EligibilityResponseTestDataFactory.java

```diff
@@ -27,6 +27,10 @@ public static EligibilityResponse anEligibilityResponseWithStatus(EligibilitySta
         return aValidEligibilityResponseBuilder().eligibilityStatus(eligibilityStatus).build();
     }
 
+    public static EligibilityResponse anEligibilityResponseWithChildrenAndStatus(List<ChildDTO> children, EligibilityStatus eligibilityStatus) {
+        return aValidEligibilityResponseBuilder().children(children).eligibilityStatus(eligibilityStatus).build();
```

Comment:
This now needs to set the voucherEntitlement to null if the eligibilityStatus != eligible